File: src/test/java/com/znsio/teswiz/screen/vodqa/WebViewScreen.java
package com.znsio.teswiz.screen.vodqa;


public abstract class WebViewScreen {

    public static WebViewScreen get() {
        return com.znsio.teswiz.screen.ScreenRegistry.getScreen(WebViewScreen.class);
    }

    public abstract boolean isUserOnNewsWebViewScreen();

    public abstract boolean isLoginOptionVisible();

    public abstract VodqaScreen navigateToSamplesList();
}

File: src/test/resources/playwright/screens/vodqa/web-view.screen.ts
import { unsupportedScreenAction } from "../screen-route.ts";

const MESSAGE = "Vodqa web view is not supported on web for WEB_ENGINE=playwright-ts.";

export function isUserOnNewsWebViewScreen(): object {
  return unsupportedScreenAction(MESSAGE);
}

export function isLoginOptionVisible(): object {
  return unsupportedScreenAction(MESSAGE);
}

export function navigateToSamplesList(): object {
  return unsupportedScreenAction(MESSAGE);
}
